generators/chipyard/src/main/scala/config/RocketConfigs.scala

```diff
@@ -212,3 +213,26 @@ class LBWIFRocketConfig extends Config(
   new freechips.rocketchip.subsystem.WithNoMemPort ++       // remove AXI4 backing memory
   new freechips.rocketchip.subsystem.WithNBigCores(1) ++
   new chipyard.config.AbstractConfig)
+
+class MulticlockAXIOverSerialConfig extends Config(
+  new chipyard.config.WithFbusToSbusCrossingType(RationalCrossing(SlowToFast)) ++
+
+  new chipyard.config.WithSystemBusFrequencyAsDefault ++
+  new chipyard.config.WithSystemBusFrequency(4000) ++
+  new chipyard.config.WithPeripheryBusFrequency(4000) ++
```

Comment:
Why 4000?

Reply:
Arbitrary. I just did this so things were nice multiples of 1000.

Reply:
If its arbitrary, can we choose realistic frequencies for these?